Better_Foliage/mod.json

```diff
@@ -0,0 +1,10 @@
+{
+  "mod_id": "betterfoliage",
+  "mod_name": "Better Foliage",
+  "mod_dir": "/BetterFoliage/",

```

Comment:
Hello,
The ``mod_dir`` need to match the dir where the texutres are stored. In this it has to be ``"mod_dir": "/Better_Foliage/",``